src/lib/common/data_getter.ts

```diff
@@ -0,0 +1,18 @@
+import type { FeatureCollection } from "geojson";
+import authoritiesUrl from "../../../assets/authorities.geojson?url";
+
+export async function getAuthoritiesData() {
```

Comment:
Can we add the return types? `: FeatureCollection`
(Hopefully we can get to enabling strict in tsconfig "soon" to catch things like this)

And how about naming it `getAuthoritiesGeojson` or `getAuthoritiesGj`? "Data" is a bit vague. The return type is helpful though (if your IDE shows it)

Reply:
Should add type, and agree GeoJson is more specific and thererfore better!